src/main/kotlin/no/nav/familie/ef/sak/repository/domain/AndelTilkjentYtelse.kt

```diff
@@ -15,14 +18,13 @@ data class AndelTilkjentYtelse(@Column("belop")
                                val personIdent: String,
                                val periodeId: Long? = null,
                                val forrigePeriodeId: Long? = null,
-                               ) {
+                               val opprinnelsesbehandlingId: UUID? = null) {
```

Comment:
Dette navnet er fint, men hvis det stemmer at BA har endret til "kilde..." kan vi jo vurdere å gjøre det samme? (for å gjøre det enklere for utviklere å hoppe mellom teamene (ba/ef))

Reply:
abbf1c3